kano_feedback/DataSender.py

```diff
@@ -357,6 +359,15 @@ def get_hdmi_info():
     return res + edid
 
 
+def get_co_list():
+    '''
```

Comment:
It may be useful to wrap this in a try, except clause just in case the issue is being caused by the Content manager